airbyte-commons-worker/build.gradle

```diff
@@ -0,0 +1,65 @@
+import org.jsonschema2pojo.SourceType
+
+plugins {
+    id "java-library"
+    id 'com.github.eirnym.js2p' version '1.0'
+}
+
+group 'io.airbyte...'
+version '0.40.14'
+
+repositories {
```

Comment:
This whole block can be removed -- it is added by the `subprojects` block of the root `build.gradle` file.

Reply:
Sorry it was an auto generate file, I removed the group and airbyte version as well.